app/views/users/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<% provide :title, t("profile.title") %>
+
+  <% if check_user @user.id %>
+    <%= link_to "<button class='btn btn-primary btn-edit'>
+      <i class='fa fa-pencil'></i> Chỉnh sửa </button>".html_safe, edit_user_path %>
+  <% end %>
+  <h3> <%= t "profile.title" %></h3>
+  <%= link_to image_tag("default-avatar.png", class: "avatar-img") %>
+  <ul>
+    <li> <b> <%= t "profile.name" %> : </b> <%= @user.full_name %> </li>
+    <li> <b> <%= t "profile.email" %> : </b> <%= @user.email %> </li>
+    <li> <b> <%= t "profile.gender" %> : </b> <%= @user.gender ? "Nam" : "Nữ" %> </li>
```

Comment:
dùng toán tử 3 ngôi ở đây để làm gì e?

Reply:
gender em để kiểu boolean, khi hiện thị ra thì hiện Nam hoặc nữ nên em dùng toán tử 3 ngôi ạ

Reply:
"Nam" : "Nữ" 
Dùng I18n em nhé